common/test/acceptance/tests/studio/test_studio_general.py

```diff
@@ -95,7 +95,7 @@ def test_page_redirect(self):
         self.dashboard_page.visit()
         self.assertEqual(self.browser.current_url.strip('/').rsplit('/')[-1], 'home')
 
-    @flaky  # TODO: FEDX-88
+    @skip  # TODO: FEDX-88
```

Comment:
Instead of doing this, please just remove AssetIndexPage from self.pages (comment out with bug number). That way we don't lose coverage on the rest of the pages.